src/components/create-study-form.js

```diff
@@ -150,6 +154,11 @@ const UploadCase = () => {
     );
 };
 
+const ghostStudyKeyGenerator = (() => {
+    let key = 1;
```

Comment:
L'équivalent d'une static interne à une fonction C ? Wow.

Reply:
i used the same mechanism as the one in merge orchestrator, generating an uuid is a bit overkill i think (need more dependency)